### PR TITLE
refactor(components): replace relative import to absolute using webpack alias

### PR DESCRIPTION
**Pull request by boiarchukk**
Replace relative import to absolute using webpack alias
## Description

For a large codebase where the developers work in parallel and move things around frequently, relative imports introduces the following problems:

1. Need to mentally traverse the directory
2. Spend time in fixing the imported modules not found

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [x] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/button/Button.stories.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import "./Button";
-import "../icon/Icon";
-import "../theme/Theme";
+import "@/components/button/Button";
```
Comment (yananym) [inline web-components/src/components/button/Button.stories.ts:1]: this is an overkill don't you think? you are literally in the same folder

File: web-components/src/components/chip/Chip.stories.ts [hunk 2]
```diff
@@ -2,8 +2,8 @@ import { withA11y } from "@storybook/addon-a11y";
 import { boolean, number, select, text, withKnobs } from "@storybook/addon-knobs";
 import { html } from "lit-html";
 import { badgeColor, BarType, iconSamples } from "@/utils/enums";
-import "../icon/Icon";
-import "./Chip";
+import "@/components/icon/Icon";
+import "@/components/chip/Chip";
```
Comment (yananym) [inline web-components/src/components/chip/Chip.stories.ts:6]: same here

File: web-components/src/components/chip/Chip.test.ts [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import "../icon/Icon";
+import "@/components/icon/Icon";
 import { Key } from "@/constants";
 import { elementUpdated, fixture, fixtureCleanup, html } from "@open-wc/testing-helpers";
-import "./Chip";
-import { Chip } from "./Chip";
+import "@/components/chip/Chip";
```
Comment (yananym) [inline web-components/src/components/chip/Chip.test.ts:4]: and here

File: web-components/src/components/combobox/ComboBox.stories.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
-import "./ComboBox";
-import "../icon/Icon";
+import "@/components/combobox/ComboBox";
```
Comment (yananym) [inline web-components/src/components/combobox/ComboBox.stories.ts:1]: and here

File: web-components/src/components/combobox/ComboBox.test.ts [hunk 5]
```diff
@@ -1,28 +1,14 @@
-import "./ComboBox";
-import { ComboBox } from "./ComboBox";
-import "../icon/Icon";
+import "@/components/combobox/ComboBox";
```
Comment (yananym) [inline web-components/src/components/combobox/ComboBox.test.ts:1]: and here

File: web-components/src/components/editable-textfield/EditableTextfield.stories.ts [hunk 6]
```diff
@@ -2,9 +2,9 @@ import { numInputTypes } from "@/utils/enums";
 import { withA11y } from "@storybook/addon-a11y";
 import { boolean, select, text, withKnobs } from "@storybook/addon-knobs";
 import { html } from "lit-html";
-import { Input } from "../input/Input";
-import "./EditableTextfield";
-import { alignment } from "./EditableTextfield";
+import { Input } from "@/components/input/Input";
+import "@/components/editable-textfield/EditableTextfield";
```
Comment (yananym) [inline web-components/src/components/editable-textfield/EditableTextfield.stories.ts:6]: and here

File: web-components/src/components/editable-textfield/EditableTextfield.test.ts [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import { elementUpdated, fixture, fixtureCleanup, html } from "@open-wc/testing-helpers";
-import { Input } from "../input/Input";
-import "./EditableTextfield";
-import { EditableTextfield } from "./EditableTextfield";
+import { Input } from "@/components/input/Input";
+import "@/components/editable-textfield/EditableTextfield";
```
Comment (yananym) [inline web-components/src/components/editable-textfield/EditableTextfield.test.ts:3]: and here

File: web-components/src/components/input/Input.stories.ts [hunk 8]
```diff
@@ -1,9 +1,17 @@
 import { withA11y } from "@storybook/addon-a11y";
 import { boolean, select, text, withKnobs } from "@storybook/addon-knobs";
 import { html } from "lit-html";
-import "../icon/Icon";
-import "./Input";
-import { containerSize, iconNames, iconPosition, inputShape, inputSize, inputType, nestedLevel } from "./Input";
+import "@/components/icon/Icon";
+import "@/components/input/Input";
```
Comment (yananym) [inline web-components/src/components/input/Input.stories.ts:5]: and here

File: web-components/src/components/label/Label.stories.ts [hunk 9]
```diff
@@ -1,8 +1,8 @@
 import { withA11y } from "@storybook/addon-a11y";
 import { text, withKnobs } from "@storybook/addon-knobs";
 import { html } from "lit-html";
-import "../input/Input";
-import "./Label";
+import "@/components/input/Input";
+import "@/components/label/Label";
```
Comment (yananym) [inline web-components/src/components/label/Label.stories.ts:5]: and here

File: web-components/src/components/meeting-alert/MeetingAlert.ts [hunk 10]
```diff
@@ -10,10 +10,10 @@ import { Key } from "@/constants";
...
 import { customElement, html, LitElement, property, TemplateResult } from "lit-element";
 import { nothing } from "lit-html";
-import "../avatar/Avatar";
-import "../button/Button";
-import "../icon/Icon";
-import { templateHTML } from "./CompositeAvatar";
+import "@/components/avatar/Avatar";
+import "@/components/button/Button";
+import "@/components/icon/Icon";
+import { templateHTML } from "@/components/meeting-alert/CompositeAvatar";
```
Comment (yananym) [inline web-components/src/components/meeting-alert/MeetingAlert.ts:16]: and here. you get the idea

**Comment by boiarchukk** [comment]
@yananym in future we can replace storybook files to another folder with different path, so personally I don't think that this is overkill.

**Review by yananym — APPROVED** [review]

**Pull request merged**